repose-aggregator/artifacts/valve/src/test/scala/org/openrepose/valve/ReposeJettySSLTest.scala

```diff
@@ -145,7 +145,7 @@ class ReposeJettySSLTest extends FunSpec with Matchers with BeforeAndAfterAll {
 
   def selectiveRequest(protocols: Array[String] = null, ciphers: Array[String] = null): Unit = {
     //Protocol names come from here:
-    //http://docs.oracle.com/javase/7/docs/technotes/guides/security/StandardNames.html#SSLContext
+    //http://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#SSLContext
```

Comment:
Did you check that link?

Reply:
Yep. I checked them all. There is one that was super weirdly changed, but it is the right one.